src/Writers/HTMLWriter.jl

```diff
@@ -29,17 +29,9 @@ the subdirectories of `assets/` are not linked).
 For the **logo**, Documenter checks for the existence of `assets/logo.png`.
 If that's present, it gets displayed in the navigation bar.
 
-For **scripts**, every `assets/*.js` gets a `<script>` link in the `<head>` tag
-of every page (except if it matches one of Documenter's default scripts;
-the filtering is done in [`user_scripts`](@ref)).
-
-Note that only javascript files are linked to the generated HTML. Any related CSS
-must be loaded by the script. With jQuery this could be done with the following
-snippet
-
-```javascript
-\$('head').append(\$('<link rel="stylesheet">').attr('href', documenterBaseURL + "/assets/<file>.css"))
-```
+Additional JS and CSS assets can be included in the generated pages using the `assets`
+keyword for `makedocs`. `assets` must be a `Vector{String}` and will include each listed
+asset in the `<head>` of every page in the order in which they are listed.
```

Comment:
Perhaps we should add something along the lines of: "the type of the asset (i.e. whether it going to be included with a `<script>` or a `<link>` tag) is determined by file's the extension?" Just to make it clear for the user.

Otherwise, looks great! :100: